sdks/python/apache_beam/runners/portability/expansion_service_test.py

```diff
@@ -298,6 +324,10 @@ def main(unused_argv):
           PipelineOptions(
               ["--experiments", "beam_fn_api", "--sdk_location", "container"])),
       server)
+  beam_artifact_api_pb2_grpc.add_ArtifactRetrievalServiceServicer_to_server(
```

Comment:
Is there a way to tag an artifact just for a given transforms instead of all transforms registered in the expansion service ? (probably future work).

Reply:
I tried in a way that you described but had a problem that popen (for fetching dependencies) didn't work well in grpc callbacks. We could try it again in the future.